.ci/Jenkinsfile

```diff
@@ -373,7 +374,6 @@ pipeline {
         allOf {
           anyOf {
             branch 'main'
-            tag pattern: 'v\\d+\\.\\d+\\.\\d+.*', comparator: 'REGEXP'
```

Comment:
Was it the intent to drop this tag pattern completely for the `Benchmarks` stage? Or to just remove the `.*`-part of the tag pattern?

Reply:
> Was it the intent to drop this tag pattern completely for the Benchmarks stage?

Yes, I wanted to run `run the release stage only for tags`, since the `Benchmarks` stage already runs on merge-commits or GitHub comments with `benchmark tests`.
Bear in mind, the builds for the tags were not available for a few months.


But if needed, I can amend my changes

Reply:
Thanks. I missed that this is in an “anyOf”.